nodejs-extract-text-webpack-plugin/nodejs-bundle-extract-text-webpack-plugin.spec

```diff
@@ -1,92 +1,85 @@
-# FIXME
-# Remove nodejs_symlink_deps if installing bundled module with NPM
-
 %global npm_name extract-text-webpack-plugin
-%global enable_tests 0
 
 %{?nodejs_find_provides_and_requires}
 
 Name: nodejs-%{npm_name}
-Version: 2.1.2
-Release: 2%{?dist}
+Version: 3.0.0
+Release: 1%{?dist}
 Summary: Extract text from bundle into a file
 License: MIT
 URL: http://github.com/webpack-contrib/extract-text-webpack-plugin
-Source0: http://registry.npmjs.org/extract-text-webpack-plugin/-/extract-text-webpack-plugin-2.1.2.tgz
+Source0: http://registry.npmjs.org/extract-text-webpack-plugin/-/extract-text-webpack-plugin-3.0.0.tgz
 Source1: http://registry.npmjs.org/schema-utils/-/schema-utils-0.3.0.tgz
 Source2: http://registry.npmjs.org/webpack-sources/-/webpack-sources-1.0.1.tgz
 Source3: http://registry.npmjs.org/loader-utils/-/loader-utils-1.1.0.tgz
-Source4: http://registry.npmjs.org/async/-/async-2.4.1.tgz
+Source4: http://registry.npmjs.org/async/-/async-2.5.0.tgz
 Source5: http://registry.npmjs.org/emojis-list/-/emojis-list-2.1.0.tgz
-Source6: http://registry.npmjs.org/json5/-/json5-0.5.1.tgz
-Source7: http://registry.npmjs.org/source-list-map/-/source-list-map-2.0.0.tgz
-Source8: http://registry.npmjs.org/big.js/-/big.js-3.1.3.tgz
+Source6: http://registry.npmjs.org/source-list-map/-/source-list-map-2.0.0.tgz
+Source7: http://registry.npmjs.org/big.js/-/big.js-3.1.3.tgz
+Source8: http://registry.npmjs.org/json5/-/json5-0.5.1.tgz
 Source9: http://registry.npmjs.org/source-map/-/source-map-0.5.6.tgz
-Source10: http://registry.npmjs.org/lodash/-/lodash-4.17.4.tgz
-Source11: http://registry.npmjs.org/ajv/-/ajv-5.2.0.tgz
-Source12: http://registry.npmjs.org/json-schema-traverse/-/json-schema-traverse-0.3.0.tgz
-Source13: http://registry.npmjs.org/fast-deep-equal/-/fast-deep-equal-0.1.0.tgz
+Source10: http://registry.npmjs.org/ajv/-/ajv-5.2.2.tgz
+Source11: http://registry.npmjs.org/lodash/-/lodash-4.17.4.tgz
+Source12: http://registry.npmjs.org/json-schema-traverse/-/json-schema-traverse-0.3.1.tgz
+Source13: http://registry.npmjs.org/fast-deep-equal/-/fast-deep-equal-1.0.0.tgz
 Source14: http://registry.npmjs.org/json-stable-stringify/-/json-stable-stringify-1.0.1.tgz
 Source15: http://registry.npmjs.org/co/-/co-4.6.0.tgz
 Source16: http://registry.npmjs.org/jsonify/-/jsonify-0.0.0.tgz
-Source17: extract-text-webpack-plugin-2.1.2-registry.npmjs.org.tgz
+Source17: extract-text-webpack-plugin-3.0.0-registry.npmjs.org.tgz
 BuildRequires: nodejs-packaging
-BuildRequires: npm
 BuildArch:  noarch
 ExclusiveArch: %{nodejs_arches} noarch
 
-Provides: npm(%{npm_name}) = %{version}
```

Comment:
Nope, it does not, we need that line or things break. I've hit this twice with updating packages using the script that removes this line.